src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUFETP

```diff
@@ -48,7 +48,7 @@
     {
       "name": "TEMP",
       "value_type": "DOUBLE",
-      "                      dimension": "Temperature"
+      "dimension": "Temperature"
```

Comment:
The temperature only be available for E300. Not sure what is the policy here, since it is E100 folder.

Reply:
That policy is not really clear - I'll merge